config/initializers/0_print_time_spent.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# This class is taken from https://github.com/nevir/Bumbler which
+# depends on bundler. But for RPM environment, Foreman does not
+# use bundler (but bundler_ext stub). The gem won't load in that
+# environment.
+module Bumbler
+  module Hooks
+    @slow_threshold = 100.0
+    @slow_items = {}
+
+    class << self
+      attr_writer :slow_threshold
+
+      attr_reader :slow_items
+
+      def benchmark(key)
+        start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+        result = yield
+        time = (Process.clock_gettime(Process::CLOCK_MONOTONIC) - start) * 1000 # ms
+        @slow_items[key] = time if time > @slow_threshold
+        [time, result]
+      end
+
+      def print_slow_items
+        Rails.logger.debug "Slow initializers:"
+        slow_items.to_a.sort_by! { |_n, t| t }.each do |name, time|
```

Comment:
Isn't this the same?
```suggestion
        slow_items.sort_by { |_n, t| t }.each do |name, time|
```

Reply:
Yeah, the method probably just returns `self` but I can fix this.

Reply:
Oh wait, `slow_items` is a hash. I fixed this.